tests/integration/io/docker-compose/retry_server/utils/request_range.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from typing import Annotated
+
+from fastapi import Header
+
+
+def parse_range_from_header(range: Annotated[str, Header()], length: int) -> tuple[int, int]:
```

Comment:
I know we were the ones who committed the sins first 😞 but we should avoid shadowing python's `range` class